tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,2 +1,32 @@
-frost-sidebar testbed
-{{frost-sidebar}}
+     {{#frost-tabs class=(if iframe 'iframe') on-change=(action 'tabSelected') selection=selectedTab}}
+  {{#frost-tab alias='Readme' id='readme'}}
```

Comment:
Thanks lot for all the hand holdings at this point. It is quite bit of learning curve.